metadata-ingestion/src/datahub/ingestion/graph/client.py

```diff
@@ -582,6 +599,36 @@ def get_urns_by_filter(
                 }
             ]
 
+        # Platform instance filter.
+        if platform_instance:
+            if platform:
+                # Massage the platform instance into a fully qualified urn, if necessary.
+                platform_instance = make_dataplatform_instance_urn(
+                    platform, platform_instance
+                )
+
+            # TODO: Warn if platform_instance is not a fully qualified urn.
+
+            andFilters += [
+                {
+                    "field": "platformInstance",
+                    "values": [platform_instance],
+                    "condition": "EQUAL",
+                }
+            ]
+
+        # Browse path v2 filter.
+        if container:
+            # TODO: Warn if container is not a fully qualified urn.
```

Comment:
These warnings would be nice... and I'd probably just fail not even warn

Reply:
added